build.gradle.kts

```diff
@@ -0,0 +1,8 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+plugins {
+    alias(libs.plugins.gradle.versions)
+    alias(libs.plugins.version.catalog.update)
```

Comment:
We should list all the plugins that the subprojects use here, with `apply false` added so that they're loaded but not applied at the root, e.g.

```
    alias(libs.plugins.android.application) apply false
```

Reply:
Thanks, didnt get why i needed this.. Done now :D